### PR TITLE
rename BusinessLogic to Interface

### PR DESCRIPTION
**Pull request by jmrodri**
Using the more Go idiomatic choice of broker.Interface.

### CODE REVIEW

**Comment by jmrodri** [comment]
Fixes issue #14.

File: pkg/rest/apisurface.go [hunk 1]
```diff
@@ -21,29 +21,29 @@ import (
 // the broker's internal business logic into the correct places in the HTTP
 // response.
 type APISurface struct {
-	// BusinessLogic contains the business logic that provides the
+	// BrokerInterface contains the business logic that provides the
 	// implementation for the different OSB API operations.
-	BusinessLogic broker.BusinessLogic
-	Metrics       *metrics.OSBMetricsCollector
+	BrokerInterface broker.Interface
```
Comment (carolynvs) [inline pkg/rest/apisurface.go]: How about naming this just `Broker`?

**Comment by jmrodri** [comment]
@LiliC I will fix the commit. I actually *hate* using `fixes 14` :)

**Comment by jmrodri** [comment]
@carolynvs will rename.

**Comment by jmrodri** [comment]
@LiliC @carolynvs done.

**Pull request merged**